sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/CreateMessageBatchOptions.cs

```diff
@@ -17,6 +17,12 @@ public class CreateMessageBatchOptions
         /// <summary>The requested maximum size to allow for the batch, in bytes.</summary>
         private long? _maxSizeInBytes;
 
+        /// <summary>
+        ///   The maximum number of messages to allow in a single batch.
+        /// </summary>
+        ///
+        internal int? MaxMessageCount { get; set; }
```

Comment:
Not sure if I'm reading incorrectly but it seems like this is always null, the default value is only stored in the AmqpSender property and this property is internal. Should AmqpMessageBatch only check the link information (currently the temp default), rather than this value?

Reply:
The flow is that `AmqpSender` reads that information, sets it on the options, and then passes it to AmqpMessageBatch, which evaluates the limits.   We could hardcode directly in the batch for the short-term, but I wanted to run the infrastructure that we'll need when the service adds the metadata.    Now, all we have to do is add two lines of code and delete two lines.